rootfs/api/urls.py

```diff
@@ -29,22 +29,6 @@
         views.ReleaseViewSet.as_view({'post': 'rollback'})),
     url(r"^apps/(?P<id>{})/releases/?".format(settings.APP_URL_REGEX),
         views.ReleaseViewSet.as_view({'get': 'list'})),
-    # restart containers
```

Comment:
They describe slightly different issues, so we should leave them both open for now IMHO.